.env.example

```diff
@@ -4,6 +4,7 @@
 #
 MYUSA_KEY: consumer_public_key
 MYUSA_SECRET: consumer_secret_key
+DEFAULT_URL_HOST: your_local_server_address
```

Comment:
sounds good! question: are you using foreman or rails server? I am using foreman bc of our multiple processes but I see a lot of localhost:3000 in setup so maybe I should just default to that...